libs/application/templates/children-residence-change/src/fields/ParentBIntro/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import { useIntl } from 'react-intl'
+import { Box, Button, Text } from '@island.is/island-ui/core'
+
+import {
+  childrenResidenceInfo,
+  formatAddress,
+  getSelectedChildrenFromExternalData,
+} from '../../lib/utils'
+import { parentBIntro } from '../../lib/messages'
+import { CRCFieldBaseProps } from '../../types'
+import { DescriptionText } from '../components'
+
+const ParentBIntro = ({ field, application }: CRCFieldBaseProps) => {
+  const { externalData, answers } = application
+  const { formatMessage } = useIntl()
+  const applicant = externalData.nationalRegistry.data
+  const childResidenceInfo = childrenResidenceInfo(applicant, answers)
+  const children = getSelectedChildrenFromExternalData(
+    applicant.children,
+    answers.selectedChildren,
+  )
+  return (
+    <>
+      <Box marginTop={3} marginBottom={5}>
+        <Text>
+          {formatMessage(parentBIntro.general.description, {
+            otherParentName: applicant.fullName,
+          })}
+        </Text>
+      </Box>
+      <Box marginTop={4} marginBottom={4}>
```

Comment:
It seems strange to have both margin top and bottom on these. I would prefer if we stick to only bottom or top here.